conjur/logic/init_logic.py

```diff
@@ -60,14 +63,27 @@ def fetch_account_from_server(conjurrc_data):
         if conjurrc_data.cert_file is None and conjurrc_data.appliance_url.startswith("https"):
             certificate_path = os.path.join(os.path.dirname(DEFAULT_CONFIG_FILE),
                                           "conjur-server.pem")
+
         else:
             certificate_path = conjurrc_data.cert_file
 
         logging.debug("Attempting to fetch the account from the Conjur server")
+
+        store = requests.certs.where()
+        with open(certificate_path) as certificate_file:
+                loaded_certificate = certificate_file.read()
+
+        # We copy the contents of the CA bundle on the machine and append to it the certificate
+        # that was fetched from the server so that certificate validation can take place.
+        # We make a duplicate of the ca bundles to avoid changing the file system of the user.
+        shutil.copy(store, DEFAULT_CERTIFICATE_BUNDLE_FILE)
+        with open(DEFAULT_CERTIFICATE_BUNDLE_FILE, 'a') as bundle:
+            bundle.write('\n'+loaded_certificate)
```

Comment:
TODO surround all requests in a try/catch to differentiate between self signed and none-self signed